src/userobjects/MoabSkinner.C

```diff
@@ -582,7 +588,7 @@ MoabSkinner::getTemperatureBin(const Elem * const elem) const
   auto value = (*_serialized_solution)(dof);
 
   // TODO: add option to truncate instead
-  if (value < _temperature_min)
+  if ((_temperature_min - value) > 1e-6)
```

Comment:
I think variables for temperature and density tolerances would be good here in place of the literals here. IMO they don't necessarily need to be exposed as user parameters (yet), but giving them names will help with clarity a bit.

Reply:
Agreed, good call